cmake/LYWrappers.cmake

```diff
@@ -310,16 +324,35 @@ function(ly_add_target)
         set_property(GLOBAL APPEND PROPERTY LY_ALL_TARGET_DIRECTORIES ${CMAKE_CURRENT_SOURCE_DIR})
     endif()
 
+    # Custom commands need to be declared in the same folder as the target that they use. 
+    # Not all the targets will require runtime dependencies, but we will generate at least an
+    # empty file for them.
     set(runtime_dependencies_list SHARED MODULE EXECUTABLE APPLICATION)
     if(NOT ly_add_target_IMPORTED AND linking_options IN_LIST runtime_dependencies_list)
 
-        add_custom_command(TARGET ${ly_add_target_NAME} POST_BUILD
+        # the stamp file will be the one that triggers the execution of the custom rule. At the end
+        # of running the copy of runtime dependencies, the stamp file is touched so the timestamp is updated.
+        # Adding a config as part of the name since the stamp file is added to the VS project.
+        # Note the STAMP_OUTPUT_FILE need to match with the one used in runtime dependencies (e.g. RuntimeDependencies_common.cmake)
+        set(STAMP_OUTPUT_FILE ${CMAKE_BINARY_DIR}/runtime_dependencies/$<CONFIG>/${ly_add_target_NAME}_$<CONFIG>.stamp)
```

Comment:
We could store off the STAMP_OUTPUT_FILE in a target_property and then just query it later in the `ly_delayed_generate_runtime_dependencies`

That way we can be sure the names match

Reply:
yeah, though about adding a global variable, but I would have to pass it with escaped variables and then generate it. And when generating it I would have to match the name of the variables
On one side is:
```
${CMAKE_BINARY_DIR}/runtime_dependencies/$<CONFIG>/${ly_add_target_NAME}_$<CONFIG>.stamp
                                                                                                 ^ needs to be configured
```
on the other side is:
```
${CMAKE_BINARY_DIR}/runtime_dependencies/$<CONFIG>/${target}_$<CONFIG>.stamp
```
And we already depend on the name of the runtime_dependencies file to match: `${CMAKE_BINARY_DIR}/runtime_dependencies/$<CONFIG>/${target}.cmake`